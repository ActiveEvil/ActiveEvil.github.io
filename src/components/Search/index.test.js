import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import Component from './';

describe('Search component', () => {
  test('renders a search input', () => {
    const wrapper = shallow(
      <Component />
    );

    console.log(wrapper);

    expect(wrapper.find('.search').exists()).toBe(true);
  });

  test('renders as expected', () => {
    const component = renderer.create(
      <Component />
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
