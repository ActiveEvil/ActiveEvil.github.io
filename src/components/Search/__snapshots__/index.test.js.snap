// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders as expected 1`] = `
<input
  aria-label="Search for Star Wars Characters..."
  className="search"
  id="search"
  name="search"
  onChange={[Function]}
  placeholder="Search for Star Wars Characters..."
  type="search"
/>
`;
